test/values.ts

```diff
@@ -1261,3 +1262,54 @@ export const mockLegacyMobileMoneyProvider: LegacyMobileMoneyProvider = {
     url: 'fake-url-1',
   },
 }
+
+export const mockNft: Nft = {
+  contractAddress: mockContractAddress,
+  media: [
+    {
+      gateway: 'https://example.com/1',
+      raw: 'https://example.com/1',
+    },
+  ],
+  metadata: {
+    attributes: [{ trait_type: 'Fizz Buzz', value: '1' }],
+    date: new Date('01/01/2020').getTime(),
+    description: 'This is a fizzBuzz name!',
+    dna: '000001',
+    id: 1,
+    image: 'https://example.com/1',
+    name: `${mockName}.fizzBuzz`,
+  },
+  ownerAddress: mockAccount,
+  tokenId: '1',
+  tokenUri: 'https://example.com/1',
+}
+
+export const mockNft2: Nft = {
+  contractAddress: mockContractAddress,
+  media: [
+    {
+      gateway: 'https://example.com/3',
+      raw: 'https://example.com/3',
+    },
+  ],
+  metadata: {
+    attributes: [{ trait_type: 'Fizz Buzz', value: 'Fizz' }],
+    date: null,
+    description: 'This is a fizzBuzz name!',
+    dna: null,
+    id: null,
+    image: 'https://example.com/3',
+    name: `${mockName}.fizzBuzz`,
+  },
+  ownerAddress: mockAccount,
+  tokenId: '3',
+  tokenUri: 'https://example.com/3',
+}
+
+export const mockNft3: Nft = {
```

Comment:
it'd be nice to name these more semantically so that it's clearer what the difference between them are...like `mockNftNoMetadata` / `mockNftPartialMetadata` or something